mmdet3d/datasets/pipelines/loading.py

```diff
@@ -238,6 +240,133 @@ def __repr__(self):
         return f'{self.__class__.__name__}(sweeps_num={self.sweeps_num})'
 
 
+@PIPELINES.register_module()
+class LoadCalibPoseTime(object):
```

Comment:
This pipeline should be placed in the get_data_info of SemanticKITTI Dataset.

Reply:
Please refer to other datasets for the way to load calibration/pose/timestamps. (We only load and process the point/img data in the pipeline.)